pkg/storage/postgres/postgres_test.go

```diff
@@ -143,3 +143,28 @@ func TestReadPageEnsureOrder(t *testing.T) {
 	require.Equal(t, firstTuple, tuples[1].Key)
 
 }
+
+func TestReadAuthorizationModelUnmarshallError(t *testing.T) {
```

Comment:
It'd be nice to see this test move into a common place and the only thing that we do differently between postgres and mysql is the `RunDatastoreTestContainer(t, dbEngine)`.